build/azDevOps/azure/ensono/ci-aeh-listener.yml

```diff
@@ -14,6 +14,13 @@ trigger:
 variables:
   - template: ci-aeh-listener-vars.yml
 
+resources:
+  repositories:
+    - repository: wiki # work around for security issue for pipelines to access the WIKI repo
+      type: git
+      name: Stacks.wiki
+      ref: wikiMaster
```

Comment:
Will this cause the pipeline to fail if the repo doesn't exist?

Reply:
You raise a good point. Yes, this will cause the pipeline to fail, however as the resources reference has been added into the `build/azdevops/azure/ensono/*` pipelines I don't believe they'll be included in any output when the CLI is used?
It does raise the question of how we make this optional in the template pipelines. I'm not sure if that can be, so it may come down to a documentation solution, i.e if you set enable_release_note_generation you then need to manually add the repository resource to each pipeline you want to enable generation for?

Reply:
Yeah that makes sense. I think having it explained in the documentation side is the best we can do